main/bind/APKBUILD

```diff
@@ -176,30 +193,57 @@ dev() {
 	mv "$pkgdir"/usr/bin/isc-config.sh "$subpkgdir"/usr/bin/
 }
 
-_py3() {
-	pkgdesc="A module allowing rndc commands to be sent from Python programs"
-	depends="$_py3deps"
+plugins() {
+	pkgdesc="The ISC DNS server plugins"
+	depends="$depends_plugins"
+
+	mkdir -p "$subpkgdir"/usr/lib
+	mv "$pkgdir"/usr/lib/bind "$subpkgdir"/usr/lib/
+}
+
+py3_isc() {
+	pkgdesc="Python functions to support BIND utilities"
+	depends="$depends_py3_isc"
+	provides="py3-isc=${_isc_ver}-r${_isc_rel} py3-$pkgname=$pkgver-r$pkgrel"
```

Comment:
Why is the versioned provides needed? I think it adds complexity for no good reason.